### ISSUE TITLE
AVT 1 - Dropdown had DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4531]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

## Detailed Description
1. Go to the React Component 
2. Run DAP and the following violation is present (see screenshot):
<img width="1091" alt="Screen Shot 2019-11-01 at 2 50 43 PM" src="https://user-images.githubusercontent.com/21676914/68052143-008d8480-fcb7-11e9-8cb4-006e106e7ed4.png">
3. Select the disable knob and run DAP again and the following violation is persent:
<img width="976" alt="Screen Shot 2019-11-01 at 2 51 54 PM" src="https://user-images.githubusercontent.com/21676914/68052265-48aca700-fcb7-11e9-846a-4fc02ac59b9a.png">
4. Deselect the disable knob and select the form invalid knob and the following violation is present:
<img width="967" alt="Screen Shot 2019-11-01 at 2 54 23 PM" src="https://user-images.githubusercontent.com/21676914/68052442-a3de9980-fcb7-11e9-99e6-789d7231c2a6.png">
